source/includes/_08-additional-features.md.erb

```diff
@@ -369,8 +377,69 @@ app.post("/search", (req, res) => {
         An object uniquely identifying this candidate from this provider,
         structured as it is in <a href="#data-structure">the PassFort API</a>
       </td>
+    </tr>
     <tr>
-  </tbody>
+      <td><code>addresses</code></td>
+      <td>
+        A list of CompanyAddress objects
+      </td>
+      <td>No</td>
+      <td>
+        Addresses of the company, with each CompanyAddress object structured
+        as it is in <a href="#data-structure">the PassFort API</a>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>contact</code></td>
+      <td>
+        A CompanyContactDetails object
+      </td>
+      <td>No</td>
+      <td>
+        Contact details for the company, in an object structured as it
+        is in <a href="#data-structure">the PassFort API</a>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>incorporation_date</code></td>
+      <td>
+        string
+      </td>
+      <td>No</td>
+      <td>
+        Exact date in YYYY-MM-DD format
+      </td>
+    </tr>
+    <tr>
+      <td><code>structure_type</code></td>
+      <td>
+        Object containing structured fields
```

Comment:
In the light of @louisdewar's impending changes https://github.com/PassFort/PassFort/pull/9061 defining StructuredCompanyType in the OpenAPI-format description of the PassFort API, we should refer to that type here.

Reply:
Done.